Project.toml

```
@@ -24,6 +24,7 @@ DataStructures = "0.18.3"
 Distributions = "0.25"
 MLJModelInterface = "1.6"
 SpecialFunctions = "0.8, 0.9, 0.10, 1, 2"
+Statistics = "1"
```

Should we follow the recommendation in https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958 and rather use
```suggestion
Statistics = "1.6"
```
?

---

It depends. Will it be possible to install an older version of a stdlib than the Julia version? That post doesn't state explicitly. My guess is no, since an older version of the stdlib may depend on a newer language feature, since they were coupled in the past. If so, then I don't see any reason to prefer one way over the other.

If specifying the same bound as the version number is preferable, it would be nice if CompatHelper did this by default.

---

These were basically my thoughts as well. I just assumed that maybe the announcement would be more trustworthy.

---

Sure, we can go that route. I'll update the PRs accordingly.